common-ui/src/main/java/com/duckduckgo/mobile/android/ui/Theming.kt

```diff
@@ -71,6 +71,7 @@ object Theming {
             "com.duckduckgo.sync.impl.ui.SyncLoginActivity",
             "com.duckduckgo.sync.impl.ui.SyncConnectActivity",
             "com.duckduckgo.sync.impl.ui.ShowQRCodeActivity",
+            "com.duckduckgo.sync.impl.ui.EnterCodeActivity",
```

Comment:
@nalcalag will need approval again for this change. Nothing new here, already approved in https://github.com/duckduckgo/Android/pull/3098, but we reverted the merge because a crash on nightlies.